diff --git a/src/google/protobuf/message_lite.h b/src/google/protobuf/message_lite.h
index 25264ef..b120b42 100755
--- a/src/google/protobuf/message_lite.h
+++ b/src/google/protobuf/message_lite.h
@@ -295,7 +295,7 @@ using MessageTraits = decltype(MessageTraitsImpl::value<T>);
 struct EnumTraitsImpl {
   struct Undefined;
   template <typename T>
-  static Undefined value;
+  static std::enable_if_t<sizeof(T) != 0, Undefined> value;
 };
 template <typename T>
 using EnumTraits = decltype(EnumTraitsImpl::value<T>);
